src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,82 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { User } from '../../types/User';
+import { actions as currentTodoActions } from '../../features/currentTodo';
+import { getUser } from '../../api';
 
 export const TodoModal: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const loadUsers = async () => {
+      try {
+        if (currentTodo) {
+          const loadedUsers = await getUser(currentTodo?.userId);
+
+          setUser(loadedUsers);
+        }
+      } catch (error) {
+        if (error) {
+          throw new Error('Users can not be loaded');
```

Comment:
Just FYI: now you throw the new Error on the error that happened during fetching a user. But you don't handle this new error at all, so it will lead to runtime error - the site will break. The correct way would be to show some error message here, in the same way as in the App.tsx